.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,18 +6,19 @@ Thank you for your submission!
 Name: <!--(to be dispayed in blog)-->
 
 
-##### Requierments for media in submissions
+#### Requirements for media in submissions:
 <!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
 
 - [ ] Images are hosted on Imgur <!--(recommended width:896px - height:504px)-->
-  - [ ] Use provided Imgur embed `{% include imgur.html id="WsUV4DK" %}` where `id` is last part of Imgur URL https://imgur.com/gallery/`WsUV4DK`
+  - [ ] Use provided Imgur embed `![](https://i.imgur.com/flSlSwB.png)`
```

Comment:
describing where id is coming from might help some people

Reply:
Ye could do, not sure exactly how to describe it tho...